### PR TITLE
ZQS-1351: Fix issues with circuits guide

### PR DESCRIPTION
**Pull request by olgOk**
## Description

- Added explanation for missing beta parameters that can cause reader's confusion.

- Changed in-line code format to code block.

- Left matrix comparison unchanged as it produces ambiguous results in different test settings.

## Please verify that you have completed the following steps

- [x] I have self-reviewed my code.
- [x] I have included test cases validating introduced feature/fix.
- [x] I have updated documentation.

### CODE REVIEW

**Comment by 1ethanhansen** [comment]
I get an error when doing `make html`:

```
reading sources... [  0%] api/integrations/cirq/conversions/_circuit_conversions/index                                                          
Exception occurred:
  File "/Users/ethan/Library/CloudStorage/OneDrive-ZapataComputing/Documents/GitHub/orquestra-core/docs/.venv/lib/python3.9/site-packages/autoapi/directives.py", line 22, in get_items
    obj = all_objects[name]
KeyError: 'integrations.cirq.conversions._circuit_conversions.NonNativeGate'
The full traceback has been saved in /var/folders/52/sskwpgjx3v3dkzxm51vgsxzc0000gn/T/sphinx-err-5n4povv3.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [html] Error 2
```

**Comment by 1ethanhansen** [comment]
Update: I removed all of the old intermediate stuff (_build, tmp_monorepo, etc) and now it works!

File: docs/quantum/guides/circuits.rst [hunk 1]
```diff
@@ -181,7 +190,14 @@ To build up the mixing circuit, we need to put parametrized RX gate on each of t
 .. note::
   ``orquestra-quantum`` comes with a `built-in utility function <https://github.com/zapatacomputing/orquestra-quantum/blob/main/src/orquestra/quantum/circuits/_generators.py>`_ that puts down a layer of the same single-qubit gate acting on all qubits.
 
-The output of that looks like ``mixing_circ: Circuit(operations=[RX(gamma_0)(0), RX(gamma_1)(1), RX(gamma_2)(2)], n_qubits=3)``. Again, for now don't worry about the ``gamma`` parameters in there, that will be addressed in the :ref:`symbolic gates <symbolic_gates>` section.
+The output of that looks like this:
+
+.. code-block:: text
+  
+  ic| mixing_circ: Circuit(operations=[RX(gamma_0)(0), RX(gamma_1)(1), RX(gamma_2)(2)], n_qubits=3)
+  Circuit(operations=[RX(gamma_0)(0), RX(gamma_1)(1), RX(gamma_2)(2)], n_qubits=3)
```
Comment (1ethanhansen) [inline docs/quantum/guides/circuits.rst:198]: looks like the output is duplicated. suggestion: leave the icecream printout, 

```suggestion
```

**Review by 1ethanhansen — CHANGES_REQUESTED** [review]
Overall good, just one suggestion 😄

File: docs/quantum/guides/circuits.rst (hunk 1 above)
Comment (olgOk) [reply docs/quantum/guides/circuits.rst:198]: right, looks like I overlooked this

**Review by 1ethanhansen — APPROVED** [review]
Builds properly on my local machine, I don't see any issues. LGTM!

**Review by jamesclark-Zapata — APPROVED** [review]

**Pull request merged**